src/main/charts/bitbucket/Changelog.md

```diff
@@ -1,5 +1,17 @@
 # Change Log
 
+
+## 1.0.0
+This is the first officially supported version of the helm chart.
```

Comment:
should `helm chart` be `Helm Charts`? If so same change throughout.

Reply:
It is probably `Helm chart` as it is a changelog for one chart? Not sure about the `C` vs `c`, @tan-ro ?

Reply:
I see we have `Helm chart` in all our documentation so I followed the existing format here.